tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
@msrubar : My only idea is to generate the OVAL file dynamically using a bash script. The temporary file will be crated by mktemp and then the file name will be put in all possible occurences in the OVAL file. Have you got a better idea?